onnxruntime/core/providers/tvm/tvm_execution_provider.cc

```diff
@@ -153,7 +162,22 @@ class TVMRunner {
         dl_tensors_inputs[counter] = t;
         inds[counter++] = i;
       }
-      tvm::TVMSetInputs(*mod_, inds, dl_tensors_inputs);
+      if (use_vm_) {
+        tvm::TVM_VM_SetInputs(*mod_, dl_tensors_inputs);
+        // Infer once for calculating of output shapes
```

Comment:
How will this affect benchmarking?

Reply:
At first run we should do probe inference to define output tensor shapes. It needs for VM only. It does not affect our benchmarking because of warmup with several inferences is done before performance test.